src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -76,22 +79,42 @@ def process_sample(self, **sample):
 
 
     def calculate_angle(self, p):
-        x1, y1 = p[0][0][0], p[0][0][1]
-        x2, y2 = p[0][1][0], p[0][1][1]
-        x3, y3 = p[1][0][0], p[1][0][1]
-        x4, y4 = p[1][1][0], p[1][1][1]
+
+        angle = 0
+
+        body_line_detected = len(p[0]) == len(p[1]) == 2
```

Comment:
@bhavikapanara please add a comment why we want this expression to be true. What does it accomplish in layman terms?
Since you are using non-descriptive variable names such as p[], the code is hard to read and maintain by others on the team without some commentary.

Reply:
Done